frontend/packages/topology/src/actions/contextMenuActions.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import { Node } from '@patternfly/react-topology';
+import { ActionsLoader } from '@console/shared';
+import ActionMenuContent from '@console/shared/src/components/actions/menu/ActionMenuContent';
+
+export const contextMenuActions = (element: Node): React.ReactElement[] => {
+  return [
+    <ActionsLoader contextId="topology-actions" scope={element}>
+      {(loader) => loader.loaded && <ActionMenuContent options={loader.options} />}
```

Comment:
Not sending an onClick handler keeps the context menu open even after the callback has been called.
<img width="1792" alt="Screenshot 2021-07-07 at 5 41 50 PM" src="https://user-images.githubusercontent.com/6041994/124757106-e2c3f880-df4a-11eb-8c02-1ad7450d869e.png">

Reply:
Changed the implementation here. PTAL